openlibrary/plugins/upstream/mybooks.py

```diff
@@ -0,0 +1,430 @@
+import json
+import web
+
+from infogami.utils import delegate
+from infogami.utils.view import safeint, render
+
+from openlibrary import accounts
+from openlibrary.core.booknotes import Booknotes
+from openlibrary.core.bookshelves import Bookshelves
+from openlibrary.core.lending import add_availability
+from openlibrary.core.observations import Observations, convert_observation_ids
+from openlibrary.core.sponsorships import get_sponsored_editions
+from openlibrary.plugins.upstream import borrow
+
+
+RESULTS_PER_PAGE = 25
+
+
+class public_my_books(delegate.page):
+    path = "/people/([^/]+)/books"
+
+    def GET(self, username):
+        raise web.seeother('/people/%s/books/want-to-read' % username)
+
+
+class public_my_books_view(delegate.page):
```

Comment:
```suggestion
class my_books_view(delegate.page):
```
Class name is misleading; some valid paths are only visible to logged in patron.